src/server/routes/uploads.js

```diff
@@ -31,11 +31,9 @@ router.post('/', requireUser, multerUpload.single('spreadsheet'), async (req, re
     console.log('Filename:', req.file.originalname, 'size:', req.file.size)
   }
 
-  const user = await getUser(req.signedCookies.userId)
```

Comment:
did you kill every instance of this pattern in the controllers? it's been pretty common, and it would be nice to consistently switch to just `const user = req.session.user`

Reply:
I think I got them all?